.python-version

```diff
@@ -0,0 +1 @@
+3.12
```

Comment:
I agree with using the highest version of Python that we support here.

Reply:
@mrobinson could you clarify your comment - is there any change needed here?

Reply:
Nope, this looks good to me!